bpf/http_sock.c

```diff
@@ -628,8 +624,8 @@ int BPF_KPROBE(kprobe_sys_exit, int status) {
         return 0;
     }
 
-    pid_key_t task = {0};
-    task_tid(&task);
+    trace_key_t task = {0};
```

Comment:
Extended data structure. It now has an extra ID, which is the current runtime ID. For now it's 0 for all other languages than NodeJS, and it's the current async_id for NodeJS.